lighthouse-core/gather/connections/connection.js

```diff
@@ -46,15 +46,16 @@ class Connection {
    * Call protocol methods
    * @param {!string} method
    * @param {!Object} params
+   * @param {Object=} options
    * @return {!Promise}
    */
-  sendCommand(method, params = {}) {
+  sendCommand(method, params = {}, options = {}) {
```

Comment:
Should also document it, not just it takes some options :) 

Wonder if it should be something like a regex so it's not just unconditionally ignoring errors. That would resemble the temporary fix we had for silencing domain enable/disable errors, which seemed like a good idea at the time